Resources/Prototypes/Reagents/medicine.yml

```diff
@@ -142,6 +142,7 @@
               Brute: -4
               Burn: -4
               Toxin: -4
+              Caustic: -4
```

Comment:
cryoxadone and siderlac are the only medicines able to heal acid damage? both are hard to use/make which makes acids overly strong

Reply:
Yeah agree to that, that's why I added it to cryoxadone in the first place and brought up the possibility of moving caustic under burn and letting ointment be used as a remedy.

On the other hand, acids are not super easy to get a hold of and they pretty strongly implicate a chemist in a crime.  It also increases the demand on cryogenic healing and/or botany's involvement in making medbay work.

So I'm somewhat torn, do you think it would be more fun to have a new, more easily produced drug for caustic damage?  How would you approach it?

Reply:
i think an advanced, but synthetic, medicine to heal acidic damage would be cool. recipe like meth where it needs to be heated to react so its hard to mass produce 300 25u pills of it

Reply:
That would be cool.  If that were added, do you think we should keep the addition to Cryoxadone?

Reply:
cryo is meant to heal everything so yeah

Reply:
Yeah, I'll create a new drug for this tomorrow and submit the commit.  How exciting.

Reply:
Sodium Carbonate and Sigynate added.  See PR details above for more info.